module/VuFind/src/VuFind/Autocomplete/SolrPrefix.php

```diff
@@ -172,7 +172,10 @@ protected function mungeQuery($query)
      */
     public function setConfig($params)
     {
-        [$this->autocompleteField, $this->facetField] = explode(':', $params, 2);
+        [$this->autocompleteField, $this->facetField, $limit] = explode(':', $params, 3);
+        if (ctype_digit($limit)) {
+            $this->limit = $limit;
+        }
```

Comment:
I began writing a unit test for SolrPrefix in #4653. It reveals that the changes here cause some warnings (PHP doesn't like the explode assignment if there is only one colon in the config string, and it doesn't like the ctype_digit check if $limit is null). I think they can be prevented with:

```suggestion
        [$this->autocompleteField, $this->facetField, $limit] = explode(':', $params . '::');
        if ($limit && ctype_digit($limit)) {
            $this->limit = $limit;
        }
```